qiskit/qpy/__init__.py

```diff
@@ -42,6 +57,17 @@
     with open('bell.qpy', 'rb') as fd:
         new_qc = qpy.load(fd)[0]
 
+the :func:`qiskit.circuit.qpy_serialization.dump` function also lets you
```

Comment:
This path is outdated - should be `qiskit.qpy.dump`. I think there's several cases throughout the documentation that need updating as well.  Do you want to do that in this PR, or in a separate one?

(This sentence also needs to start with a capital letter.)

Reply:
I'll fix it now, it's easy to do and it'll be good to get this in the same backport PR.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9212/commits/c07be51deb127505a5a6a9641bae9e45139aaba2